libs/sdk-ui-geo/src/core/geoChart/GeoChartLegendRenderer.tsx

```diff
@@ -202,8 +257,37 @@ function renderPushpinSizeLegend(
     const {
         size: { data, format, name },
     } = geoData;
+    const width = contentRect?.client?.width;
+    const isSmall =
```

Comment:
consider helpers for these predicates

Reply:
Done, check it please: https://github.com/gooddata/gooddata-ui-sdk/pull/1341/commits/8448bfe3f1903efa6639c4f50a462137021a94e9